scripts/jenkins/groovy/makeTarget.groovy

```diff
@@ -108,7 +108,7 @@ def call(final pipelineContext, final Closure body) {
 
 private void execMake(final String buildAction, final String h2o3dir) {
   sh """
-    export JAVA_HOME=`find /usr/lib/jvm -name '*java*${env.JAVA_VERSION}*' -type l`
+    export JAVA_HOME=`find /usr/lib/jvm -name '*java*${env.JAVA_VERSION}*' -type l | head -n1`
```

Comment:
`r3.5-init-java-11` failed with the following cryptic message.
```
[2024-01-29T07:40:28.678Z] > 
[2024-01-29T07:40:28.678Z] > #Call h2o.init() just in case instance is not running
[2024-01-29T07:40:28.678Z] > h2o.init(jvm_custom_args = jvm_custom_args)
[2024-01-29T07:40:28.678Z] 
[2024-01-29T07:40:28.678Z] H2O is not running yet, starting it now...
[2024-01-29T07:40:28.678Z] <simpleError in system2(command, "-version", stdout = TRUE, stderr = TRUE): error in running command>
[2024-01-29T07:40:28.678Z] Error in value[[3L]](cond) : 
[2024-01-29T07:40:28.678Z]   You have a 32-bit version of Java. H2O works best with 64-bit Java.
[2024-01-29T07:40:28.678Z] Please download the latest Java SE JDK from the following URL:
[2024-01-29T07:40:28.678Z] https://www.oracle.com/technetwork/java/javase/downloads/index.html
[2024-01-29T07:40:28.678Z] Calls: h2o.init ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
[2024-01-29T07:40:28.678Z] Execution halted
[2024-01-29T07:40:28.678Z] make: *** [scripts/jenkins/Makefile.jenkins:194: test-r-init] Error 1
```

This was caused by having a multiline `JAVA_HOME`:
```
JAVA_HOME='/usr/lib/jvm/java-11-oracle
/usr/lib/jvm/java-1.11.0-openjdk-amd64'
```